res/layout/activity_manage_space.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+              android:orientation="vertical"
+              android:layout_width="match_parent"
+              android:layout_height="match_parent">
+
+    <TextView
+        android:id="@+id/general_description"
+        android:layout_height="wrap_content"
+        android:layout_width="match_parent"
+        android:layout_gravity="center_vertical"
+        android:singleLine="false"
+        android:layout_margin="10dp"
```

Comment:
we do have a value for that already called standard_margin (actually 16dp) and the same for padding (standard_padding) both are defined according to the material standards for margins and paddings.